src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,24 @@
+package pages;
+
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+    @Contract("_ -> new")
+    private @NotNull By addToCartButtonByProduct(int productId) {
```

Comment:
are we going to use this method anywhere else than in `addProductToCart` method? maybe there's no need to have this code in a separate method :)

Reply:
I would leave it as it is for now. 
- easier test isolation. 
- this method may will use in other places when i prepare divide home page separate modules
- better readability